contribs/src/main/java/com/netflix/conductor/contribs/listener/DynoQueueStatusPublisher.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ *
+ */
+package com.netflix.conductor.contribs.listener;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.common.run.WorkflowSummary;
+import com.netflix.conductor.core.config.Configuration;
+import com.netflix.conductor.core.events.queue.Message;
+import com.netflix.conductor.core.execution.WorkflowStatusListener;
+import com.netflix.conductor.dao.QueueDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.Collections;
+
+/**
+ * Publishes a @see Message containing a @see WorkflowSummary to a DynoQueue on a workflow completion or termination event.
+ */
+public class DynoQueueStatusPublisher implements WorkflowStatusListener {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DynoQueueStatusPublisher.class);
+    private final QueueDAO queueDAO;
+    private final ObjectMapper objectMapper;
+    private final Configuration config;
+    private final String successStatusQueue;
+    private final String failureStatusQueue;
+
+    @Inject
+    public DynoQueueStatusPublisher(QueueDAO queueDAO, ObjectMapper objectMapper, Configuration config) {
+        this.queueDAO = queueDAO;
+        this.objectMapper = objectMapper;
+        this.config = config;
+        this.successStatusQueue = config.getProperty("status.publisher.success.queue", "_callbackSuccessQueue");
```

Comment:
I'd suggest to make queue name property self explanatory. Something along the lines of `workflowstatuslistener.publisher.success.queue`.

Reply:
nice, I guess it can't get more self explanatory than this